staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -200,10 +200,27 @@ type GroupResources struct {
 	// The empty string represents the core API group.
 	// +optional
 	Group string
-	// Resources is a list of resources within the API group.
-	// Any empty list implies every resource kind in the API group.
+	// Resources is a list of resources within the API group. Specific subresources
+	// can be matched using a "/" to indicate the subresource. For example "pods" matches
+	// requests to "pods" and "pods/logs" but "pods/logs" only matches "pods/logs".
+	//
+	// To ignore specific subresources, policies should match the subresource before the
+	// non-subresourced kind.
+	//
+	//		policy:
+	//		- level: None
+	// 		  resources: ["pods/logs"] # Ignore pods/logs.
+	// 		- level: Metadata
+	//		  resources: ["pods"] # Catch all for "pods" and all other pods subresources.
```

Comment:
Nit
the policy example is invalid, It should be:
```
- level: None
  resources: ["pods/log"] # Ignore pods/log.
- level: Metadata
   resources: ["pods"] # Catch all for "pods" and all other pods subresources.
```

1. a middle level "resources" is missed
2. subresource is "log" rather than "logs"

We  also need to update the example in github page, I think.
https://github.com/kubernetes/kubernetes/pull/48836#issue-242523068